### PR TITLE
Remove FMUX and JMU related code

### PR DESCRIPTION
**Pull request by filip-stenstrom**
The FMUX and JMU model formats are obsolete. This PR removes related code.

### CODE REVIEW

File: src/common/xmlparser.py [hunk 1]
```diff
@@ -1,5718 +0,0 @@
-#!/usr/bin/env python
```
Comment (PeterMeisrimelModelon) [inline src/common/xmlparser.py:1]: If we remove this file, the reference to it in src/common/__init__.py + references in the documentation generation should also be removed.

Comment (filip-stenstrom) [reply src/common/xmlparser.py:1]: Will do. As part of building it's apparently copied to pyjmi.common.xmlparser and pymodelica.common.xmlparser, so I might need to revert this tho.

File: src/common/core.py [hunk 2]
```diff
@@ -449,7 +449,7 @@ def get_unit_name(class_name, unit_type='FMU'):
             The name of the model.
             
         unit_type --
```
Comment (PeterMeisrimelModelon) [inline src/common/core.py:451]: It looks like one could actually remove the whole `get_unit_name `function, couldn't find any place it is used.

**Review by PeterMeisrimelModelon — APPROVED** [review]
Looks great, happy that we can remove some obsolete parts.

**Pull request merged**